README.md

```diff
@@ -317,6 +300,6 @@ HIVE is currently maintained by Nick Reinicke ([@nreinicke] (https://github.com/
 
 ## Notice
 
-Copyright © 2022 Alliance for Sustainable Energy, LLC, Inc. All Rights Reserved
+Copyright © 2023 Alliance for Sustainable Energy, LLC, Inc. All Rights Reserved
```

Comment:
i _think_ this copyright year shouldn't be carried forward, it should stay fixed at 2022 to coincide with the [software record](https://www.osti.gov/biblio/1896384)

Reply:
That makes sense, I'll revert it